cmd/ssm-run/README.md

```diff
@@ -40,42 +40,48 @@ If a region is not specified, you will receive the following warning, but execut
 WARNING	No AWS region setting found. Will default to the region linked to any profile in use.
 ```
 
-The use of the `--limit` flag, if set, will result in your command being executed on the specified number of random instances returned for each profile/region combination. For example, if you set `--limit=10` on the account `foo` for regions `us-east-1` and `us-west-2`, your command would be executed on up to 20 total instances.
-
 #### running command(s) on a single instance
 
 ```
-> ./ssm-run -i i-12345 -c "uname -a"
-
-INFO    Command(s) to be executed: uname -a
-INFO    Fetched 1 instances for account [profile1] in [us-east-1].
-INFO    Instance ID              Region          Profile         Status
-INFO    i-12345					 us-east-1       profile1         Success
+> ./ssm run -p 'profile1' -i i-12345 -c 'uname'
```

Comment:
Is there a reason to keep the ./ssm in? Feel like that was an artifact from when this was bash script or the binary was written to the root of the project. A local build of the project would be using bin/ssm and an installed version using 'go get' would not need the './' prefix. This comment applies to many other examples.